src/OpenApi/Schemas/Resources/Post/PostModelResourceSchema.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace CSlant\Blog\Api\OpenApi\Schemas\Resources\Post;
+
+use CSlant\Blog\Api\OpenApi\Schemas\Models\CategorySchema;
+use CSlant\Blog\Api\OpenApi\Schemas\Resources\Author\AuthorModelResourceSchema;
+use CSlant\Blog\Api\OpenApi\Schemas\Resources\Tag\TagModelResourceSchema;
+use OpenApi\Attributes\Items;
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
+#[Schema(
+    schema: "PostModelResource",
+    required: ["id", "name", "slug", "description", "categories", "tags", "author"],
+    properties: [
```

Comment:
@pxthinh please follow all params in this resource `packages/blog-api-package/src/Http/Resources/PostResource.php`